### ISSUE TITLE
Plot loss

### ISSUE THREAD

**Issue by Alexa201299** [taipingeric/yolo-v4-tf.keras#24]
How can I plot training and validation loss curves? And in which code should i write it?
